otx/cli/builder/supported_backbone/mmdet.json

```diff
@@ -14,36 +14,60 @@
           "regnetx_12gf"
         ]
       },
-      "available": ["ATSS", "YOLOX", "Segmentation", "Mask-RCNN"]
+      "available": [
```

Comment:
How about unifying all JSON files into single file? (i.e. `otx/cli/builder/supported_backbones.json`)
I'm not sure whether the location of JSON files that has a dependency on frameworks is good or not since they are located under the `otx/cli` and `builder` that could be represented such a general term.

Reply:
There are parts that I think this is also a design choice. Agreeing with some of the comments, the downside to keeping it in one file is that it doesn't have good visibility. One file becomes very long and I suspect that if there is a problem with that file it could affect the whole framework's backbone find and build.

Reply:
Yeah, it can differ from person to person. Let's leave it as is.